domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/vilkår/kvp/KVPVilkår.kt

```diff
@@ -0,0 +1,82 @@
+package no.nav.tiltakspenger.saksbehandling.domene.vilkår.kvp
+
+import no.nav.tiltakspenger.libs.periodisering.Periode
+import no.nav.tiltakspenger.libs.periodisering.PeriodeMedVerdi
+import no.nav.tiltakspenger.libs.periodisering.Periodisering
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.Lovreferanse
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.SkalErstatteVilkår
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.Utfall
+
+/**
+ * Kvalifiseringsprogrammet (KVP): https://www.nav.no/kvalifiseringsprogrammet
+ *
+ * @param søknadSaksopplysning Saksopplysninger som kan være avgjørende for vurderingen. Kan ikke ha hull. [avklartSaksopplysning]/faktumet er den avgjørende saksopplysningen.
+ * @param avklartSaksopplysning Faktumet som avgjør om vilkåret er oppfylt eller ikke. Null implisiserer uavklart.
+ * @param utfall Selvom om utfallet er
+ *
+ */
+data class KVPVilkår private constructor(
+    val søknadSaksopplysning: KvpSaksopplysning,
+    val saksbehandlerSaksopplysning: KvpSaksopplysning?,
+    val avklartSaksopplysning: KvpSaksopplysning,
+    val utfall: Periodisering<Utfall>?,
```

Comment:
Trenger denne å være nullable? Til nå har vi defaultet til UAVKLART for hele vurderingsperioden, og det har fungert fint.

Reply:
Gjør om denne til ikke-nullable. Lager Utfall2 som skal erstatte Utfall. OPPFYLT,IKKE_OPPFYLT,UAVKLART. Vurderer senere eller aldri om vi trenger en TotaleUtfall som også inneholder DELVIS_UAVKLART.